Ginger/Ginger/packages.config

```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AvalonEdit" version="5.0.4" targetFramework="net461" />
+  <package id="CommandLineParser" version="2.6.0" targetFramework="net461" />
```

Comment:
where and for what this CommandLineParser is been used in Ginger project?

Reply:
It is not, but without the ref the build was missing the lib for Ginger.exe so I added ref